sf-pcap.c

```diff
@@ -794,6 +777,7 @@ pcap_setup_dump(pcap_t *p, int linktype, FILE *f, const char *fname)
 pcap_dumper_t *
 pcap_dump_open(pcap_t *p, const char *fname)
 {
+	const pcap_ioplugin_t *plugin = pcap_ioplugin_init(getenv("PCAP_IOPLUGIN_WRITE"));
```

Comment:
Presumably your other pull request will eventually allow a program using libpcap to choose to use different I/O routines, so that, for example, a program could provide a command-line flag, or a GUI checkbox, or something such as that to do compressed output, rather than requiring the user to set an environment variable.